### ISSUE TITLE
Please provide a valid WRITE query

### ISSUE THREAD

**Issue by moxious** [neo4j-contrib/neo4j-spark-connector#242]
<img width="707" alt="Screen Shot 2020-11-04 at 7 18 20 AM" src="https://user-images.githubusercontent.com/5471710/98110998-f6740800-1e6d-11eb-985d-843d6ce0a495.png">

I'm a bit confused on what the rules are for a valid write query

I think we need a FAQ entry for "please provide a valid (READ|WRITE) query" describing the criteria.

**Comment by smithna** [comment]
I agree. Would like to see example of best practice for constraint/index creation for a new environment as in demo above.

**Comment by cybersam** [comment]
At the very least, this error message should include the same error message that the neo4j browser would show.

**Comment by ali-ince** [comment]
We had done some improvements around error messages over the years.
